chrome/content/zotero/bibliography.js

```diff
@@ -87,10 +85,18 @@ var Zotero_File_Interface_Bibliography = new function() {
 			selectIndex = 0;
 		}
 		
+		// add locales to list
+		if(!_io.locale) {
+			_io.locale = Zotero.Prefs.get("export.lastLocale");
```

Comment:
Doesn't make it a good idea.

On Wed, May 6, 2015 at 8:31 PM, Rintze M. Zelle <notifications@github.com>
wrote:

> In chrome/content/zotero/bibliography.js
> <https://github.com/zotero/zotero/pull/623#discussion_r29818514>:
>
> > @@ -87,10 +85,18 @@ var Zotero_File_Interface_Bibliography = new function() {
> >  			selectIndex = 0;
> >  		}
> >  		
> > +		// add locales to list
> > +		if(!_io.locale) {
> > +			_io.locale = Zotero.Prefs.get("export.lastLocale");
>
> But the existing code does the same for the style:
>
>         if(!_io.style) {
>             _io.style = Zotero.Prefs.get("export.lastStyle");
>             _saveStyle = true;
>         }
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zotero/zotero/pull/623/files#r29818514>.
>